### PR TITLE
Fix bugs for v1.4

### PR DESCRIPTION
**Pull request by clydelhui**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -301,13 +302,27 @@ Examples:
...
 
 Displays the time slots in a week when all members of the specified group are free (no events).
 
-Format: free GROUP_INDEX [START_DATE]
+Format: `free GROUP_INDEX [START_DATE]`
 
-* Finds unoccupied time slots within members of the group at the specified `INDEX`. The index refers to the index number
+* Finds unoccupied time slots within members of the group at the specified `GROUP_INDEX`. The index refers to the index number
   shown in the displayed group list. The index *must be a positive* 1, 2, 3 ...
 * The `START_DATE` field is optional. The time slots will be shown for the week ahead, starting from that date. If it is
```
Comment (Alvozyn) [inline docs/UserGuide.md:327]: Can you help add that the START_DATE should be in the format of `dd/MM/yyyy`? Thanks!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -301,13 +302,27 @@ Examples:
 * `re_edit 1 1 d/Tuesday`
 * `re_edit 1 1 t/14:00`
 
+### Export a person: `export`
+Export a person's details from the address book. 
+
+Format: `export [INDEX_OF_PERSON]`
+
+* Exports a person's details with the specified `INDEX`
```
Comment (Alvozyn) [inline docs/UserGuide.md:310]: I think maybe the `INDEX` can be changed into `INDEX_OF_PERSON` just to be more clear,

**Review by Alvozyn — COMMENTED** [review]
Just a minor change other than that lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (clydelhui) [reply docs/UserGuide.md:327]: Yups! Stated in line 329

File: docs/UserGuide.md (hunk 2 above)
Comment (clydelhui) [reply docs/UserGuide.md]: Thanks for the suggestion! Added!

**Pull request merged**